code/_onclick/item_attack.dm

```diff
@@ -267,14 +273,14 @@
 	SEND_SIGNAL(src, COMSIG_ITEM_POST_ATTACK_ATOM, attacked_atom, user)
 
 /// Called from [/obj/item/proc/attack_atom] and [/obj/item/proc/attack] if the attack succeeds
-/atom/proc/attacked_by(obj/item/attacking_item, mob/living/user)
+/atom/proc/attacked_by(obj/item/attacking_item, mob/living/user, crit)
```

Comment:
if this didn't require piping into this proc I would fullmerge this behind a holiday flag

Reply:
I was trying to think how I'd do that but I just couldn't figure it out